db/migrate/20230614111942_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,7 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+    add_column :posts, :address, :string
+    add_index :posts, :address
```

Comment:
- [ ] Please remove these columns as they are not part of the project's requirement and they will not be used in this project. Implementing this change will keep things simple while implementing the project's requirements. 👍 💻